File: Dataset/Leetcode/valid/66/632.py
class Solution:
    def XXX(self, digits: List[int]) -> List[int]:
        a = -1
        for b in digits:
            a += 1
        digits[a] += 1
        while True:
            for c in digits:
                if c == 10:
                    d = digits.index(10)
                    digits[d] = 0
                    d -= 1
                    if d == -1:
                        digits.insert(0, 1)
                        break
                    else:
                        digits[d] += 1

            if max(digits) != 10:
                return digits
                

File: Dataset/Leetcode/train/11/632.cpp
 class Solution {
public:
    int XXX(vector<int>& height) {
        int res = 0, l = 0, r = height.size() - 1;
        while (l < r) res = max(res, (r - l) * height[height[l] <height[r] ? l++ : r--]);
        return res;
    }
};

